frame/gilt/src/mock.rs

```diff
@@ -98,12 +99,21 @@ ord_parameter_types! {
 	pub const One: u64 = 1;
 }
 
+pub struct IgnoredIssuance;
+impl Get<u64> for IgnoredIssuance {
+	fn get() -> u64 {
+		// Account zero is ignored.
+		Balances::total_balance(&0)
+	}
+}
+
```

Comment:
```suggestion
pub struct IgnoredIssuance;
impl Get<u64> for IgnoredIssuance {
	fn get() -> u64 {
		// Account zero is ignored.
		Balances::total_balance(&0)
	}
}
```

fyi you can replace all of this by a line `static IgnoredIssuance: u64 = 0` inside `parameter_types!`.